app/views/overview/_list-row-actions.html

```diff
@@ -15,7 +15,11 @@
               <a href="" ng-click="row.startBuild(row.buildConfigs[0])">Start Build</a>
             </li>
             <li ng-if="'deploymentconfigs/instantiate' | canI : 'create'" role="menuitem">
-              <a href="" ng-click="row.startDeployment()">Deploy</a>
+              <a
+                href="" 
+                ng-attr-aria-disabled="{{canDeploy() ? undefined : 'true'}}"
+                ng-class="{ 'disabled-link': !canDeploy() }"
```

Comment:
I don't think adding the class actually prevents you from clicking the menu item. Won't that still cause an error if you click the disabled item?